bundler/helpers/v2/monkey_patches/definition_ruby_version_patch.rb

```diff
@@ -10,7 +10,7 @@ def source_requirements
         Gem::Specification.new("Ruby\0", requested_version)
     end
 
-    %w(2.5.3 2.6.10 2.7.7 3.0.5 3.2.0).each do |version|
+    %w(2.5.3 2.6.10 2.7.7 3.0.5 3.2.0 3.2.1).each do |version|
```

Comment:
We normally keeping just the latest patch version of each minor release, so I'd follow that pattern.

An exhaustive list would be more correct but nobody pins to exact rubies through library dependencies, and applications pinning to exact rubies are already handled by the lines above this one.

Reply:
Thanks 🙂